code/data_structures/src/linked_list/LinkedList.hs

```diff
@@ -0,0 +1,52 @@
+{-/* Part of Cosmos by OpenGenus Foundation */-}
```

Comment:
Why not just use single line comments? (`--`)

Reply:
I though that it is suppose to be block comments, as `/*` is for block comments in C. Sorry about that. Thanks for reviewing.

Reply:
Sure, but no need to apologize! This isn't a mistake, more like a preferred style by the community here.